ui-admin/src/forms/designer/utils/formDraftUtils.tsx

```diff
@@ -0,0 +1,42 @@
+import { VersionedForm } from '@juniper/ui-core'
+
+export type FormDraft = {
+  content: string
+  date: number
+}
+
+/** returns a form draft key for local storage */
+export function getFormDraftKey({ form }: { form: VersionedForm }) {
+  return `surveyDraft_${form.id}_${form.version}`
+}
+
```

Comment:
I like how you've pulled all these functions out so it will be easier when we eventually move to having drafts saved server-side